src/StockForm.js

```diff
@@ -34,15 +39,50 @@ class StockForm extends React.Component {
         this.sendData();
     }
 
+    /**
+     * Sets the displayed graph to the mode clicked by the user
+     */
+
+    handleIntradayClick() {
+        this.setState({
+            graphMode: "Today's "
+        }, () => {
+            this.updateScreen();
+        })
+    }
+
+    handleDailyClick() {
+        this.setState({
+            graphMode: "100 days of "
+        }, () => {
+            this.updateScreen();
+        })
+    }
+
+    updateScreen() {
```

Comment:
I'm not the biggest fan of calling this updateScreen. I think using the word 'Screen' is a bit ambiguous. I think in this case you can probably be very specific. Like calling it renderNewData() (and obviously changing the two calls to it above too)

Reply:
Done